leaderboard.js

```diff
@@ -251,10 +268,20 @@ function emptyLeaderboard(msg) {
 
 }
 
+function findMember(message, id) {
```

Comment:
Can you add a few comments to findMember and findUser to explain what these do?

Reply:
Added these comments